ui/src/app/edge/live/evcs/evcs.component.ts

```diff
@@ -61,9 +61,41 @@ export class EvcsComponent {
     });
   }
 
-  ngOnDestroy() {
-    if (this.edge != null) {
-      this.edge.unsubscribeChannels(this.websocket, EvcsComponent.SELECTOR + this.componentId);
+  /**
+   * Returns the state of the EVCS
+   * 
+   * @param state 
+   * @param plug 
+   * 
+   */
+  getState(state: number, plug: number) {
+    if (this.controller.properties.enabledCharging != null && this.controller.properties.enabledCharging == false) {
+      return this.translate.instant('Edge.Index.Widgets.EVCS.ChargingStationDeactivated');
+    }
+    let chargeState = state;
+    let chargePlug = plug;
+
+    if (chargePlug == null) {
+      return this.translate.instant('Edge.Index.Widgets.EVCS.NotCharging');
+    } else if (chargePlug != ChargePlug.PLUGGED_ON_EVCS_AND_ON_EV_AND_LOCKED) {
+      return this.translate.instant('Edge.Index.Widgets.EVCS.CableNotConnected');
+    }
+    switch (chargeState) {
+      case ChargeState.STARTING:
```

Comment:
Versucht hier immer die gleichen Begriffe zu verwenden, dann findet man sich in der translate-Datei leichter zurecht (z. B. "READY_FOR_CHARGING" -> "CarFull")